specification/maps/resource-manager/Microsoft.Maps/preview/2020-04-01-preview/maps-management.json

```diff
@@ -0,0 +1,1143 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Azure Maps Resource Provider",
+    "description": "Resource Provider",
+    "version": "2020-04-01-preview",
```

Comment:
>2020-04-01-preview [](start = 16, length = 18)

this should be closer to the current date.  It is odd to have an api-version that is in the future

Reply:
@pilor, if I change this do you know if there will be any side effects from the open api hub or this PR?

Reply:
I don't know about openapihub but the PR will be fine